main.py

```diff
@@ -0,0 +1,83 @@
+from flask import Flask, request, jsonify, abort
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+import os
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = "mysql+mysqlconnector://vlada:VLVL@localhost/iot-test-db"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+ma = Marshmallow(app)
+
+
+class Door(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    ser_num = db.Column(db.Integer)
+    material = db.Column(db.String(40))
+
+    def __init__(self, ser_num, material):
+        self.ser_num = ser_num
+        self.material = material
+
+
+class DoorSchema(ma.Schema):
+    class Meta:
+        fields = ('id', 'ser_num', 'material')
+
+
+door_schema = DoorSchema()
+doors_schema = DoorSchema(many=True)
+
+
+@app.route("/door", methods=["POST"])
+def add_door():
+    ser_num = request.json['ser_num']
+    material = request.json['material']
```

Comment:
handle case with incorrect input json (e.g. typo in field name) - you shouldn't get 500 response